src/Tribe/iCal.php

```diff
@@ -203,8 +210,7 @@ public static function generate_ical_feed( $post = null ) {
 		$blogName    = get_bloginfo( 'name' );
 
 		if ( $post ) {
-			$events_posts   = array();
-			$events_posts[] = $post;
+			$events_posts = is_array( $post ) ? $post : array( $post );
```

Comment:
wouldn't the cast convert the post object to an array rather than creating an array with the post as the first element?  seems like different behavior